_docs/concepts/volumes.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: guide
+title: StorageOS Docs - Volumes
+anchor: concepts
+module: concepts/volumes
+---
+
+# Volumes
+
+StorageOS volumes are a logical construct which represent a writeable volume
+and exhibit standard POSIX semantics. We present volumes as mounts into
+containers via the StorageOS FUSE layer.
```

Comment:
I don't think we use FUSE for that! (It's worth checking, but FUSE was really slow when we tried to substitute NBD, being LIO the winning option)

"We present volumes as mounts into containers via the StorageOS FUSE layer" --> We present volumes as mounts into containers via the StorageOS LIO (Linux IO) target layer. 

Linux IO being the open source implementation of the SCSI target